module/device/method/wsa.py

```diff
@@ -145,5 +145,35 @@ def get_display_id(self):
 
     @retry
     def display_resize_wsa(self, display):
-        logger.warning('display ' + str(display) + ' should be resized')
-        self.adb_shell(['wm', 'size', '1280x720', '-d', str(display)])
+        import win32gui
+        import ctypes
+        import win32con
+        import win32api
+        def winEnumHandler(hwnd, ctx):
+            if win32gui.IsWindowVisible(hwnd):
+                n = win32gui.GetWindowText(hwnd)
+                s = win32api.GetWindowLong(hwnd, win32con.GWL_STYLE)
+                x = win32api.GetWindowLong(hwnd, win32con.GWL_EXSTYLE)
+                if n:
+                    allvisiblewindows.update({n: {'hwnd': hwnd, 'STYLE': s, 'EXSTYLE': x}})
+        allvisiblewindows = {}
+        hwnd = 0
+        multi_server_name = ['Azur Lane', '碧蓝航线', 'アズールレーン', '碧藍航線']
+        win32gui.EnumWindows(winEnumHandler, None)
+        for i in multi_server_name:
+            if i in allvisiblewindows:
+                hwnd = allvisiblewindows.get(i).get('hwnd')
+                break
```

Comment:
I would like to ask why not use `FindWindow` here?

Reply:
sure, updated